packages/api/src/libs/queries.ts

```diff
@@ -51,6 +54,13 @@ export default (appState: Dependencies) => {
     // [timestamp, price], returns just price
     return synthPrices.latest[empAddress][1];
   }
+  async function getLatestMarketPrice(address: string, currency: "usdc" = "usdc") {
```

Comment:
how do decimals work with this exactly? is it always returned in the decimals of the `currency`?

Reply:
prices are returned as a float and on return i convert it to we

Reply:
right buy what are the decimals? i.e lets say the price is 1000 USDC per eth. is it represented as "1000" or is it "1000e6"  using the decimals for USDC?